packages/app/src/app.ts

```diff
@@ -365,15 +365,15 @@ export class App<Req extends Request = Request, Res extends Response = Response>
       await applyHandler<Req, Res>(handler as unknown as Handler<Req, Res>)(req, res, next)
     }
 
-    const matchFilter = (mw: Middleware[]) => (x: Middleware) =>
+    const matchFilter = (x: Middleware) =>
       (req.method === 'HEAD' || (x.method ? x.method === req.method : true)) && !mw.includes(x)
 
     let idx = 0
 
     const loop = () => {
       req.originalUrl = req.baseUrl + req.url
       const pathname = getPathname(req.url)
-      const matched = this.#find(pathname).filter(matchFilter(mw))
+      const matched = this.#find(pathname).filter(matchFilter)
```

Comment:
if this is only used once, it can be inlined

Reply:
Agree, done